quixstreams/dataframe/registry.py

```diff
@@ -13,19 +13,19 @@
     from .dataframe import StreamingDataFrame
 
 
-class DataframeRegistry:
+class DataFrameRegistry:
     """
     Helps manage multiple `StreamingDataFrames` (multi-topic `Applications`)
     and their respective repartitions.
 
-    `SDF`s are registered by storing their topic and current Stream.
     """
 
     def __init__(self) -> None:
-        self._registry: Dict[str, Stream] = {}
-        self._topics: List[Topic] = []
-        # {repartition_topic_name: source_topic_name}
-        self._repartition_origins: Dict[str, str] = {}
+        self._registry: dict[str, Stream] = {}
+        self._topics: list[Topic] = []
```

Comment:
We are left with a single usage of `Dict` and `List` in this module. Let's finish the conversion and remove imports.